### PR TITLE
change symbol for ERC20 Margin Tokens

### LINKED ISSUE #444: Use correct symbols for short / leveraged tokens
We should use correct symbols for short and leveraged tokens. For short it should be: `"s" + owedToken.symbol()` and for leveraged it should be `"l" + heldToken.symbol()`

### PR DESCRIPTION
**Pull request by BrendanChou**
fixes #444 

`l[UNINITIALIZED]` , `dYdX Leveraged Long Token [UNINITIALIZED]`
`s[UNINITIALIZED]` , `dYdX Short Token [UNINITIALIZED]`

Initialized Longs:
`lETH`
`dYdX Leveraged Long Token 0x0d1d4e623d10f9fba5db95830f7d3839406c6af20d1d4e623d10f9fba5db9583`

Initialized Shorts
`sETH`
`dYdX Short Token 0x0d1d4e623d10f9fba5db95830f7d3839406c6af20d1d4e623d10f9fba5db9583`

### CODE REVIEW

File: contracts/lib/StringHelpers.sol [hunk 1]
```diff
@@ -58,7 +58,7 @@ library StringHelpers {
     }
 
     /**
-     * Translates a bytes32 into an ascii hexadecimal representation of those bytes
+     * Translates a bytes32 into an ascii hexadecimal representation of those bytes "0x..."
```
Comment (AntonioJuliano) [inline contracts/lib/StringHelpers.sol]: "0x..." isn't super clear to me. Does this mean the returned string will always start with "0x"?

File: contracts/lib/StringHelpers.sol [hunk 2]
```diff
@@ -70,8 +70,10 @@ library StringHelpers {
         pure
         returns (bytes)
     {
-        bytes memory numberAsString = new bytes(64);
         uint256 number = uint256(input);
+        bytes memory numberAsString = new bytes(66);
```
Comment (AntonioJuliano) [inline contracts/lib/StringHelpers.sol]: Can you map out in a comment what all of these 66 bytes are used for?

File: contracts/lib/StringHelpers.sol [hunk 3]
```diff
@@ -70,8 +70,10 @@ library StringHelpers {
...
+        numberAsString[0] = byte(48);  // '0'
```
Comment (AntonioJuliano) [inline contracts/lib/StringHelpers.sol]: Why'd you change the behavior of this to prepend "0x"?

File: contracts/margin/external/ERC20/ERC20Position.sol [hunk 4]
```diff
@@ -153,16 +149,14 @@ contract ERC20Position is
...
-        string _symbol
+        address[] trustedWithdrawers
     )
         public
         OnlyMargin(margin)
     {
         POSITION_ID = positionId;
         state = State.UNINITIALIZED;
         INITIAL_TOKEN_HOLDER = initialTokenHolder;
-        symbol = _symbol;
```
Comment (AntonioJuliano) [inline contracts/margin/external/ERC20/ERC20Position.sol]: this should save some deploy gas too 😄

File: contracts/lib/StringHelpers.sol (hunk 3 above)
Comment (AntonioJuliano) [reply contracts/lib/StringHelpers.sol]: Also want to confirm this isn't used by anything except the margin tokens

Comment (BrendanChou) [reply contracts/lib/StringHelpers.sol]: It is only used for the margin tokens

File: contracts/margin/external/ERC20/ERC20Position.sol (hunk 4 above)
Comment (BrendanChou) [reply contracts/margin/external/ERC20/ERC20Position.sol]: Yeah I'm still not sure what the gas cost is per byte of code

File: contracts/lib/StringHelpers.sol (hunk 1 above)
Comment (BrendanChou) [reply contracts/lib/StringHelpers.sol]: yeah

File: contracts/margin/external/ERC20/ERC20Position.sol (hunk 4 above)
Comment (AntonioJuliano) [reply contracts/margin/external/ERC20/ERC20Position.sol]: well it does an SSTORE so likely 20k

Comment (AntonioJuliano) [reply contracts/margin/external/ERC20/ERC20Position.sol]: think it stores that variable in storage, not code

**Pull request merged**